src/DERC20.sol

```diff
@@ -247,23 +247,34 @@ contract DERC20 is ERC20, ERC20Votes, ERC20Permit, Ownable {
 
     /**
      * @notice Releases `amount` of vested tokens
-     * @param amount Amount of tokens to release
+     * @param requestedAmount Amount of tokens to release
      */
     function release(
-        uint256 amount
+        uint256 requestedAmount
```

Comment:
I think we can just scrap the requestedAmount arg and transfer `availableAmount`?